src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -3487,6 +3487,14 @@ def create_lb_frontend_ip_configuration(
     return get_property(poller.result().frontend_ip_configurations, item_name)
 
 
+def update_lb_frontend_ip_configuration_setter(cmd, resource_group_name, load_balancer_name, parameters, gateway_lb):
```

Comment:
Are there any tests covering cross-tenant scenarios?

Reply:
Current tests already covered aux_subscriptions=[] and aux_subscriptions=[xxx] for single tenant. Cross tenant will be tested manually by service team as gateway_lb is not public accessible now.